users/models.py

```diff
@@ -45,3 +42,14 @@ def save(self, *args, **kwargs):
         super(Profile, self).save(*args, **kwargs)
 
 
+class DateTimeModel(models.Model):
+
+    datetime = models.DateTimeField()
+    is_utc = models.BooleanField(default=False)
+
```

Comment:
This has 2 problems.

1. Even if you know that you need to start converting pst to utc or vice verca the count will again be started from 0
2. I dont think it needs 2 table for this. it does not need such normalization

simply using count Field inside your model will serve the purpose i.e.
you are running command and was at 14 number you stopped it and want to resume back, you already have the info about if the existing cycle is of is_utc or is_pst you can simply continue with your cycle and reset once it is on 100.